### DISCUSSION TITLE
filtered_feature_bc_matrix.h5

### DISCUSSION THREAD

**Discussion by saramoein372** [tuonglab/dandelion#113] [General]
Hi Kelvin,

After saying thanks, I have a question: where "filtered_feature_bc_matrix.h5" file is generated? Is it from cell ranger? What is the format of this file? What are rows and what are columns?

Thanks,
Sara

**Comment by zktuong** [comment]
Yes it’s a cellranger output file. You will have to figure out the file format yourself as it’s not a dandelion specific problem.
